api/rest/restapi.go

```diff
@@ -421,6 +421,12 @@ func (api *API) routes() []route {
 			"/pins/{keyType:ipfs|ipns|ipld}/{path:.*}",
 			api.unpinPathHandler,
 		},
+		{
+			"RepoGC",
+			"POST",
+			"/repo/gc",
```

Comment:
`/peers/gc or peers/ipfs/gc.` (move up next to the `peers` endpoints).

Reply:
Didn't we agree on not linking this with peers and using `ipfs gc` https://github.com/ipfs/ipfs-cluster/pull/739#discussion_r278049953

It's nice to have your commands and APIs matching. We have done that so far. If we are using command as `ctl ipfs gc` the API should be probably be `/ipfs/gc`

Reply:
I also said that I did not like `/repo/gc`...

Thing is, `/ipfs/` is not a resource type (yet). A peer is a resource, and `POST /peers/ipfs` would indicate that we want to run an `/ipfs` action on our peers. That's my logic.

But `/ipfs/gc` might also be ok if we consider `/ipfs` will at some point be a resource (for example to return IPFSID objects directly), although IPFS ID objects are more a sub-resource of peers. I think this is slightly more human-friendly though.

Btw, it's not true commands match API endpoints. Pin add/rm talk to `/allocations` while `status` talks to `/pins`. Traditionally commands strived to keep familiarity while API strived to be consistent with data types and objects in cluster.

Reply:
I have changed this to `/ipfs/gc`. Is that fine?